SUPPORT.md

```diff
@@ -0,0 +1,15 @@
+# Support
+
+## How to file issues and get help
+
+This project has been created to accelerate the development of Confidential Consortium Framework (CCF) by Microsoft and its partners while working with customers.
+
+This project does not have a dedicated team of maintainers outside of those who are working on an active engagements and as such issues will be responded to on a best efforts basis. No guarantees can be offered as to response times on issues, feature requests, or to the long term road map for the project.
+
+Taking the above statement into account the team involved in this effort wish for the project to evolve into a stable, production ready resource with active contributors from Microsoft and the wider community.
```

Comment:
That also does not seem right, these are samples, the goal is not for the project to evolve into production-ready software. I think we should re-phrase this document to accurately describe the fact that this is a set of samples, provided as learning resources for users wanting to build on ACL. Community contributions are welcome, but are not particularly a goal.